doc/book/app_server/luajit_memprof.rst

```diff
@@ -514,15 +552,42 @@ the profiling report becomes much prettier:
 ..  code-block:: console
 
     ALLOCATIONS
-    @format_concat.lua:9, line 10: 110       2131    0
-    @format_concat.lua:0, line 0: 4 640     0
-    INTERNAL: 3     1148    0
-
+    @format_concat.lua:10: 109 events	+2112 bytes	-0 bytes
+    @format_concat.lua:0: 4 events	+640 bytes	-0 bytes
+    INTERNAL: 3 events	+1206 bytes	-0 bytes
+    
     REALLOCATIONS
-
+    
     DEALLOCATIONS
-    INTERNAL: 113   0       2469
-            Overrides:
-                    @format_concat.lua:0, line 0
-                    @format_concat.lua:9, line 10
-                    INTERNAL
+    INTERNAL: 112 events	+0 bytes	-2460 bytes
+    	Overrides:
+    		@format_concat.lua:0
+    		@format_concat.lua:10
+    		INTERNAL
+    
+    
+    HEAP SUMMARY:
+    INTERNAL holds 1144 bytes: 3 allocs, 1 frees
+    @format_concat.lua:0 holds 384 bytes: 4 allocs, 1 frees
+
+.. _luajit_memprof_leak_only:
+
+The heap summary and the --leak-only option
+-------------------------------------------
+
+This feature was added in version :doc:`2.8.1 </release/2.8.1>`.
+
+The end of each display is a HEAP SUMMARY section which looks like this:
+
+@<filename>:<line number> holds <number of still reachable bytes> bytes: <number of allocation events> allocs, <number of deallocation events> frees
+
+Sometimes, especially when there are many ``collectgarbage()`` calls,
```

Comment:
It's not only caused by `collectgarbage()` call. When there are **really tons** of allocated and freed objects (for example, if a program runs a lot of time, or GC pressure is really high for a chunk to profile). When it happens, a profiler output becomes unreadable, if we want to understand what "lines of code" (i.e. Lua objects) still holds Lua memory.